cli/cmd/plugin/standalone-cluster/create.go

```diff
@@ -43,9 +45,17 @@ func init() {
 	CreateCmd.Flags().StringVarP(&iso.infrastructureProvider, "infrastructure", "i", "", "Infrastructure to deploy the standalone cluster on. Only needed when using -i docker.")
 	CreateCmd.Flags().StringVarP(&iso.bind, "bind", "b", "127.0.0.1:8080", "Specify the IP and port to bind the Kickstart UI against (e.g. 127.0.0.1:8080).")
 	CreateCmd.Flags().StringVarP(&iso.browser, "browser", "", "", "Specify the browser to open the Kickstart UI on. Use 'none' for no browser. Defaults to OS default browser. Supported: ['chrome', 'firefox', 'safari', 'ie', 'edge', 'none']")
+	CreateCmd.Flags().DurationVarP(&iso.timeout, "timeout", "t", constants.DefaultLongRunningOperationTimeout, "Time duration to wait for an operation before timeout. Timeout duration in hours(h)/minutes(m)/seconds(s) units or as some combination of them (e.g. 2h, 30m, 2h30m10s)")
 }
 
 func create(cmd *cobra.Command, args []string) error {
+	fmt.Print("\n!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n")
```

Comment:
@jpmcb can you update this deprecation notice updated to mention the use of `unmanaged-cluster` since it will ship alongside `standalone-cluster`

Reply:
@joshrosso - how's this wording look?

![Screen Shot 2022-01-10 at 11 06 07 AM](https://user-images.githubusercontent.com/23109390/148816372-1ad9170c-efbc-4b11-a4f3-2900c445eacd.png)

Reply:
FYI - https://github.com/vmware-tanzu/community-edition/pull/2795

Reply:
love it. going to make one small recommendation that I missed (will do it in that PR.